R/check.R

```diff
@@ -2,12 +2,14 @@
 #' Check an R package on r-hub
 #'
 #' @param path Path to a directory containing an R package, or path to
-#'   source R package tarball built with `R CMD check`.
+#'   source R package tarball built with `R CMD build` or \link{build}.
```

Comment:
This must be `[devtools::build()]`. (`rhub` uses the new Markdown feature of the dev version of roxygen2.)